### PR TITLE
base-files: add migration information for APK

### PR DESCRIPTION
**Pull request by aparcar**
If the `apk` package manager is installed, show a table of common commands to deal with the system.

### CODE REVIEW

**Comment by efahl** [comment]
I might add a caveat at the bottom regarding use of "upgrade" (or maybe leave it out???), as we've long discouraged people from doing a blind `opkg upgrade` due to the risk of breaking a shady ABI-versioned library.

**Comment by hnyman** [comment]
Please start with apk update, as that is usually needed first (before apk add etc.)

**Comment by ynezz** [comment]
> Please start with apk update, as that is usually needed first (before apk add etc.)

`apk add` will do the update of the feeds when needed, no need to do it explicitly.

**Review by ynezz — APPROVED** [review]
LGTM, thanks!

File: package/base-files/files/etc/profile [hunk 1]
```diff
@@ -38,3 +38,21 @@ in order to prevent unauthorized SSH logins.
...
+ ------------------------------------------------------------------
+ opkg install <pkg>     apk add <pkg>       Install a package
+ opkg remove <pkg>      apk del <pkg>       Remove a package
+ opkg upgrade           apk upgrade         Upgrade all packages
+ opkg files <pkg>       apk info -L <pkg>   List package contents
+ opkg list-installed    apk info            List installed packages
+ opkg update            apk update          Update package lists
+ opkg search <pkg>      apk search <pkg>    Search for packages
+ ------------------------------------------------------------------
+EOF
```
Comment (ynezz) [inline package/base-files/files/etc/profile:57]:
```suggestion

For more https://openwrt.org/docs/guide-user/additional-software/opkg-to-apk-cheatsheet

EOF
```

**Comment by aparcar** [comment]
Please feel free to merge I'm not on my computer anytime soon

**Comment by Ansuel** [comment]
@aparcar ok to include changes from @ynezz ?

**Comment by hnyman** [comment]
Maybe the spelling mistake should be corrected...

OpenWrt recently switched to the "apk" package **manger!**
-->
OpenWrt recently switched to the "apk" package manager!

**Comment by Ansuel** [comment]
>:(

**Comment by efahl** [comment]
> OpenWrt recently switched to the "apk" package **manger!**

I thought it should be mangler.

**Comment by raenye** [comment]
Maybe also add a warning about `apk upgrade` -- this upgrades all packages without confirmation. Caught me by surprise.

**Comment by ynezz** [comment]
> Maybe also add a warning about `apk upgrade` -- this upgrades all packages without confirmation. Caught me by surprise.

@raenye You can propose improvements via pull request :)

BTW since this is still ongoing effort, we've added link to https://openwrt.org/docs/guide-user/additional-software/opkg-to-apk-cheatsheet where you can help make this clear. There is https://openwrt.org/docs/guide-user/additional-software/apk which needs some work as well.

**Pull request merged**